doc/cci_guide.md

```diff
@@ -0,0 +1,1703 @@
+
+    //Closing the request handle
+    error = cci_close_req_handle (req);
+    if (error < 0)
+    {
+        printf ("close_req_handle error: %d, %s\n", cci_error.err_code,
+                cci_error.err_msg);
+        goto handle_error;
+    }
+
+    //Disconnecting with the server
+    error = cci_disconnect (con, &cci_error);
+    if (error < 0)
+    {
+        printf ("error: %d, %s\n", cci_error.err_code, cci_error.err_msg);
+        goto handle_error;
+    }
+
+    return 0;
+
+handle_error:
+    if (req > 0)
+        cci_close_req_handle (req);
+    if (con > 0)
+    cci_disconnect (con, &cci_error);
+
+    return 1;
+}
+
+```
+### Supplement
+#### include head file
+below is the base
+```C
+#include <stdio.h> //print and so on
+#include "cas_cci.h"  //CCI API
+```
+but you can also add others
+```C
+#include <string.h>
+#include <stdio.h>
+#include <sys/time.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include "cas_cci.h"
+```
+>Note: you need to refer to C/C++ programing.
+
+#### variables definition
+##### within the main function:
+```C
+int con = 0, req = 0, col_count = 0, i, ind;
+int error;
+char *data;
+T_CCI_ERROR cci_error;    
+T_CCI_COL_INFO *col_info;
+T_CCI_CUBRID_STMT stmt_type; 
+char *query = "select * from code"; 
+```
+The T_CCI_ERROR object is most commonly defined. It is used in the CCI API.   
+We need to print cci_error.err_msg and cci_error.err_code, for example: 
+```C
+cci_execute(req,0,0,&cci_error)
+```
+##### outside the main function:
+```C
+char user[32]   = "dba";
+char passwd[32] = "";
+char *dbname    = NULL;
+char *url       = "cci:cubrid:xseed080.kdev:33888:bug_12530:dba::";
+T_CCI_DATASOURCE *dataSource = NULL;
+char sql[SQL_MAXNUM][56] = {"drop table if exists t1",
+        "create table t1 (a int)"};
+```
+`user`, `passwod` and url can be used in `cci_connect_with_url(url, user, passwd)`
+
+#### CCI API 
+* Opening a database connection handle (related functions: cci_connect(), cci_connect_with_url())   
+* Getting the request handle for the prepared statement (related function: cci_prepare())     
+* Binding data to the prepared statement (related function: cci_bind_param())    
+* Executing the prepared statement (related function: cci_execute())    
+* Processing the execution result (related functions: cci_cursor(), cci_fetch(), cci_get_data(), cci_get_result_info())     
+* Closing the request handle (related function: cci_close_req_handle())   
+* Closing the database connection handle (related function: cci_disconnect())     
+* Using database connection pool (related functions: cci_property_create(), cci_property_destroy(), cci_property_set(), cci_datasource_create(), cci_datasource_destroy(), cci_datasource_borrow(), cci_datasource_release(), cci_datasource_change_property())     
+
+## 6.2 CCI API reference
+Please refer to [CUBRID manual](https://www.cubrid.org/manual/en/10.1/api/cciapi.html)     
+for example:
+![cci_execute](./cci_image/CCI_API.png)
+
+## 6.3 Execute cci case alone  
```

Comment:
Move 6.3 to chapter 2.

Needn't describe in detail. It's okay to follow way in https://github.com/CUBRID/cubrid-testtools/blob/95124dfa69ff905401096ec0b9cf783ee818da12/doc/shell_heavy_guide.md#25-execute-one-test-case

Reply:
after introduce cci case,  how to debug error in test cases ,it will be understood.
it is not the same with  https://github.com/CUBRID/cubrid-testtools/blob/95124dfa69ff905401096ec0b9cf783ee818da12/doc/shell_heavy_guide.md#25-execute-one-test-case
it is has nothing to do with CTP, the only association is common functions in $init_path/init.sh